content/de/es6.md

```diff
@@ -1,5 +1,81 @@
-<!-- Pending translation -->
- <meta http-equiv="refresh" content="0;/en/es6.html">
- <script>window.location.href='/en/es6.html';</script>
+# ES6 und io.js
 
-<!-- de i18n team, feel free to put a redirect notice here -->
+Die io.js-Entwicklung basiert auf der aktuellen Version der [V8](https://code.google.com/p/v8/)-Engine. Mit der Bestrebung, ständig die aktuellste Version der V8-Engine einzusetzen, stellen wir sicher, dass neue Funktionen der [JavaScript ECMA-262 Spezifikation](http://www.ecma-international.org/publications/standards/Ecma-262.htm) sowie Geschwindigkeits- und Stabilitätsverbesserungen schnell für io.js-Entwickler zu Verfügung stehen.
+
+Die Version {{project.current_version}} io.js wird mit der V8-Version {{project.current_v8}} ausgeliefert, welche bereits ES6-Funktionen beinhaltet, die deutlich weiterentwickelter sind, als jene der Version 3.28.73, die in joyent/node@0.12.x enthalten sind.
+
+## Überflüssiger Startparameter `--harmony`
+
+In joyent/node@0.12.x (V8 3.28+) aktiviert der Startparameter `--harmony` alle ES6-Funktionen auf einmal. Enthalten sind alle jene Funktionen, die als **completed** (Fertiggestellt), **staged** (Auslieferungsvorbereitung) und **in progress** (Entwicklungsphase) gekennzeichnet sind. (Mit der Ausnahme des _nonstandard/non-harmonious_ Ausdrucks `typeof`, welcher sich hinter dem Startparameter `--harmony-typeof` versteckt). Das bedeutet, dass einige sehr fehlerhafte oder sogar kaputte Funktionen wie [proxies](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy) genauso frei verfügbar sind, wie solche Funktionen wie [generators](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function*), die nur sehr wenige Fehler enthalten, wenn überhaupt. Daraus hätte sich die Best-Practice ergeben die entweder nur bestimmte Funktionen durch einzelne Startparameter verfügbar machen (z.B. `--harmony-generators`), oder einfach alle Funktionen einschalten, aber dann nur eine sehr restriktive Untermenge an Funktionen zu verwenden.
```

Comment:
Please check the last sentence again, I think it should rather be:
"Daraus hätte sich die Best-Practice ergeben, entweder nur bestimmte Funktionen durch einzelne Startparameter verfügbar zu machen (z.B. `--harmony-generators`), oder einfach alle Funktionen einzuschalten, aber dann nur eine sehr restriktive Untermenge an Funktionen zu verwenden."